iodata/formats/qchemlog.py

```diff
@@ -0,0 +1,366 @@
+
+
+@document_load_one("qchemlog", ['atcoords', 'athessian', 'atmasses', 'atnums', 'charge', 'energy',
+                                'g_rot', 'mo', 'lot', 'nelec', 'obasis_name', 'run_type', 'extra'])
+def load_one(lit: LineIterator) -> dict:
+    """Do not edit this docstring. It will be overwritten."""
+    data = load_qchemlog_low(lit)
+    result_labels = ['atcoords', 'atmasses', 'atnums', 'charge',
+                     'charge', 'energy', 'g_rot', 'run_type']
+    result = {label: data.get(label, None) for label in result_labels}
+    # build molecular orbitals
+    # todo: double check if this is right
+    # mo_energies
+    # restricted case
+    if not data['unrestricted']:
+        mo_energies = np.concatenate(
+            (data['alpha_mo_occupied'], data['alpha_mo_unoccupied']), axis=0)
+        mo_coeffs = np.empty((data['nbasis'], data['norba'])) * np.nan
+        mo_occs = np.zeros(mo_coeffs.shape[1])
+        mo_occs[:data['alpha_elec']] = 1.0
+        mo_occs[:data['beta_elec']] += 1.0
+        mo = MolecularOrbitals("restricted", data['norba'], data['norba'],
+                               mo_occs, mo_coeffs, mo_energies, None)
+    # unrestricted case
+    else:
+        mo_energies = np.concatenate(
+            (data['alpha_mo_occupied'], data['alpha_mo_unoccupied'],
+             data['beta_mo_occupied'], data['beta_mo_unoccupied']), axis=0)
+        mo_coeffs = np.empty((data['nbasis'], data['norba'] + data['norbb'])) * np.nan
+        mo_occs = np.zeros(mo_coeffs.shape[1])
+        # number of alpha & beta electrons and number of alpha molecular orbitals
+        na, nb = data['alpha_elec'], data['beta_elec']
+        na_mo = len(data['alpha_mo_occupied']) + len(data['alpha_mo_unoccupied'])
+        mo_occs[:na] = 1.0
+        mo_occs[na_mo: na_mo + nb] = 1.0
+        mo = MolecularOrbitals("unrestricted", data['norba'], data['norbb'],
+                               mo_occs, mo_coeffs, mo_energies, None)
+    result['mo'] = mo
+
+    result['lot'] = data['method']
+    result['nelec'] = data['alpha_elec'] + data['beta_elec']
+    result['obasis_name'] = data['basis_set'].lower()
+    # moments
+    moments_labels = ['dipole_moment', 'quadrupole_moments', 'dipole_tol']
+    moments = {label: data.get(label, None) for label in moments_labels}
+    # extra information
+    extra_labels = ['spin_multi', 'nuclear_repulsion_energy', 'nbasis', 'charge',
+                    'mulliken_charges', 'polarizability_tensor', 'hessian',
+                    'imaginary_freq', 'vib_energy', 'entropy_dict']
+    extra = {label: data.get(label, None) for label in extra_labels}
+    # unit conversions for vibrational energy
+    extra['vib_energy'] = extra.get('vib_energy') * kcalmol
+    # convert kcal/mol to atomic units
+    enthalpy_dict = {k: v * kcalmol for k, v in data['enthalpy_dict'].items()}
+    # todo: unit conversions for entropy
+    extra['enthalpy_dict'] = enthalpy_dict
+    extra['moments'] = moments
```

Comment:
The moments should be properly stored in `result["moments"]` dictionary, see https://github.com/theochem/iodata/blob/master/iodata/iodata.py#L140

Reply:
Thanks for noticing this. They have been fixed in 1bb64ee and 16071c7.